adal/build.gradle

```diff
@@ -46,6 +46,8 @@ android {
             manifest.srcFile 'src/main/AndroidManifest.xml'
             java.srcDirs = ['src/main/java', 'src/main/aidl']
         }
+
+        androidTest.setRoot('src/androidTest')
```

Comment:
This is where the tests are, I'm just including them to enable automated CI scenarios.